src/v2/Apps/Order/Routes/Status/index.tsx

```diff
@@ -148,10 +148,10 @@ export class StatusRoute extends Component<StatusProps> {
         // accepted
         return this.canceledOfferOrderCopy()
       default:
-        // This should not happen. Check the order states are all accounted for:
+        // This should not happen. Check the order displayState here:
         // https://github.com/artsy/exchange/blob/master/app/models/order.rb
         // (Aside from PENDING and ABANDONED)
-        logger.error(`Unhandled order state: ${state}`)
+        logger.error(`Unhandled order state: ${displayState} in ${state} state`)
```

Comment:
Should this be:
```suggestion
        logger.error(`Unhandled order state: ${displayState} in ${displayState} state`)
```

Generally a little confused by this comment. Why are we ignoring `Pending` and `Abandoned`?

Reply:
@annacarey - no, I intentionally left the state in there as well as this should never be shown to users and if for some reason that is - it would be helpful for us to know both State and DisplayState for easier debugging.

Pending and Abandoned should be filtered way before and never shown to users in this view.

Reply:
Got it thank you! This is good to go for me!